Model/Behavior/AuditableBehavior.php

```diff
@@ -302,6 +299,8 @@ public function afterDelete(Model $Model) {
 		$this->Audit = ClassRegistry::init('Audit');
 		$this->Audit->create();
 		$this->Audit->save($data);
+
+		$this->_unsetOriginalDataForModel($Model);
```

Comment:
Also unsets the original data on deletes now.
Not sure, if that is actually wanted.